apps/ledger-live-mobile/src/components/Nft/NftList.tsx

```diff
@@ -1,16 +1,42 @@
-import React, { useCallback } from "react";
+import React, { useCallback, useMemo } from "react";
 import { FlatListProps } from "react-native";
-import { NFTMetadata, ProtoNFT } from "@ledgerhq/types-live";
-import { Flex } from "@ledgerhq/native-ui";
+import { ProtoNFT } from "@ledgerhq/types-live";
+import { Button, Flex, Text } from "@ledgerhq/native-ui";
 import { BigNumber } from "bignumber.js";
-import { useNavigation } from "@react-navigation/native";
-
+import { Stop } from "react-native-svg";
+import styled, { useTheme } from "styled-components/native";
+import Animated, { FadeInDown, FadeOutDown } from "react-native-reanimated";
 import NftListItem from "./NftListItem";
 import { AddNewItem } from "./AddNewItemList";
 import CollapsibleHeaderFlatList from "../WalletTab/CollapsibleHeaderFlatList";
 import globalSyncRefreshControl from "../globalSyncRefreshControl";
-import { track, TrackScreen } from "../../analytics";
-import { NavigatorName, ScreenName } from "../../const";
+import { TrackScreen } from "../../analytics";
+import { useNftList } from "./NftList.hook";
+import BackgroundGradient from "../TabBar/BackgroundGradient";
+
+const darkGradients = [
+  {
+    height: 145,
+    opacity: 1,
+    stops: [
+      <Stop key="0%" offset="0%" stopOpacity={0} stopColor="#131214" />,
```

Comment:
Here you can use the color background.main from the DS, it's the same one. That way you don't need two gradients anymore

Reply:
Yes indeed ;)